docs/en-US/component/statistic.md

```diff
@@ -11,13 +11,13 @@ Display statistics.
 
 :::demo To highlight a number or a group of numbers, such as statistical value, amount, and ranking, you can add elements such as icon and unit before and after the number and title.
 
-statistic/num
+statistic/basic
 
 :::
 
 ## Countdown
 
-:::demo Set `timeIndices`Start the countdown. Countdown component, support English and Chinese countdown, support to add other components control countdown.
+:::demo Countdown component, support English and Chinese countdown, support to add other components control countdown.
```

Comment:
`support English and Chinese countdown` 
`Countdown` don't seem to care what language it is.

Reply:
updated.